modules/script/script-impl/src/test/resources/myapplication/inner-script.js

```diff
@@ -0,0 +1,7 @@
+console.log("Inner script");
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692791)
```suggestion
console.log('Inner script');
```